src/main/java/com/philips/research/bombase/ConfigProperties.java

```diff
@@ -10,13 +10,23 @@
 @ConfigurationProperties(prefix = "bom-base")
 public class ConfigProperties {
     private boolean scanLicenses = true;
+    private boolean useClearlyDefined = true;
 
     public boolean isScanLicenses() {
         return scanLicenses;
     }
+    
+    public boolean isUseClearlyDefined() {
```

Comment:
Is this method about "is" or "use" ? 

(My suggestion is to make the name better express the intention.)

Reply:
Changed it into `harvestClearlyDefined()`